src/java.base/share/classes/java/lang/Object.java

```diff
@@ -478,6 +478,12 @@ public final void wait(long timeoutMillis, int nanos) throws InterruptedExceptio
      * A subclass overrides the {@code finalize} method to dispose of
      * system resources or to perform other cleanup.
      * <p>
+     * <b>When running in a Java virtual machine in which finalization has been
```

Comment:
Disabling finalization is not part of the Java SE spec.   This paragraph describes the implementation of HotSpot VM and I think it does not belong to this javadoc.

Reply:
The coupling between this change and [8276422](https://bugs.openjdk.java.net/browse/JDK-8276422) ("Add command-line option to disable finalization") is probably tighter than would be ideal.  That [CSR](https://bugs.openjdk.java.net/browse/JDK-8276773) allows for finalization to be disabled in the SE Platform Spec and the JLS.

Reply:
Thanks for pointing out that the CSR for JDK-8276422 ("Add command-line option to disable finalization") includes the change of the SE Platform Spec and JLS to allow an implementation for finalization to be disabled.   This makes senses now.